session/session.go

```diff
@@ -0,0 +1,113 @@
+package session
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"launchpad.net/gozk"
+)
+
+var (
+	ErrZkSessionNotConnected       = fmt.Errorf("Unable to connect to ZooKeeper.")
```

Comment:
Go style guides would suggest `Zk` should be `ZK` in all cases. Golint would complain about that if it knew "zk" is an abbreviation.